app/js/views/services-list-item.jsx

```diff
@@ -41,6 +49,14 @@ export default class ServicesListItem extends React.Component {
     }
   }
 
+  componentWillUnmount() {
+    switch (this.service.type) {
+      case 'motion-sensor':
+        this.service.off('motion', this.onMotion);
```

Comment:
> Can we do something like this.service.offAll('motion') here?

Mmm, imo it's a bad practice, we should not remove event listeners we didn't set on the object we don't own. Especially when service is placed in a global pool and who knows what other subview/service/controller/etc set its own listeners.

> make sure we don't forget to unlisten to anything.

Well, if this is case - then we have bug in our code :) And we should investigate and fix it rather than masking :)

What do you think?